docs/MO_DG/prepare_model/convert_model/Converting_Model.md

```diff
@@ -4,6 +4,11 @@ With model conversion API you can increase your model's efficiency by providing
 
 @sphinxdirective
 
+.. meta:: 
+   :description: When provided an additional shape definition with "input_shape" 
```

Comment:
if we want to make call-to-action descriptions, this could also be something like learn how to increase the efficiency of a model with MO by providing an additional shape definition with the "input_shape" and "static_shape" parameters.